lib/blogs_api/user/login.ex

```diff
@@ -0,0 +1,16 @@
+defmodule BlogsApi.User.Login do
+  @moduledoc """
+  Vai até o banco de dados e procura pelo email que corresponde ao email da requisição
+  """
+  alias BlogsApi.{User, Repo}
+
+  def get_by_email(email) do
+    case Repo.get_by(User, email: email) do
+      nil ->
+        {:error, :bad_request}
+
+      user ->
+        {:ok, user}
+    end
```

Comment:
Já que a requisição está correta, mas o usuário não existe, talvez esse erro poderia ser `404 - not found`?

Sobre o erro `400 - bad request`:
 > The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).

Sobre o erro `404 - not found`:
> The server cannot find the requested resource. In the browser, this means the URL is not recognized. **In an API, this can also mean that the endpoint is valid but the resource itself does not exist.** (...)

[Referência](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses)

```suggestion
  def get_by_email(email) do
    case Repo.get_by(User, email: email) do
      nil ->
        {:error, :not_found}

      user ->
        {:ok, user}
    end
```

Reply:
Oi Helena, eu concordo com você. Mas só fiz isso pra seguir o padrão dos requisitos do projeto... :)